From d8566028d5e71612d011ff06bb9f4d03d1391e8e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?J=C3=B6rg=20Thalheim?= <joerg@thalheim.io>
Date: Sun, 22 Sep 2019 21:06:54 +0100
Subject: [PATCH 24/27] Revert "sysinit.target: Drop the dependency on
 local-fs.target and swap.target"

This reverts commit 5b0b3d004beb45687347ad01fa3b99635d1bdd92.

We removed that patch in our 242 branch, but somehow it landed in 243 back again.
---
 units/sysinit.target | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/units/sysinit.target b/units/sysinit.target
index 5fe9ad0ec9..b6c16a1412 100644
--- a/units/sysinit.target
+++ b/units/sysinit.target
@@ -11,4 +11,5 @@
 Description=System Initialization
 Documentation=man:systemd.special(7)
 Conflicts=emergency.service emergency.target
-After=emergency.service emergency.target
+Wants=local-fs.target swap.target
+After=local-fs.target swap.target emergency.service emergency.target
-- 
2.24.1

